README.md

```diff
@@ -79,7 +79,7 @@ The table below lists the supported auto auto-remediate functions:
 |  Config | [Config-001](https://www.cloudconformity.com/conformity-rules/Config/aws-config-enabled.html)    | Ensure AWS Config is enabled in all regions |
 |  S3  | [S3-001](https://www.cloudconformity.com/conformity-rules/S3/s3-bucket-public-read-access.html) | Ensure S3 buckets do not allow public READ access |
 |  RDS | [RDS-008](https://www.cloudconformity.com/conformity-rules/RDS/rds-publicly-accessible.html)    | Ensure RDS instances are not public facing to minimise security risks |
-|  CloudTrail | [CT-001](https://www.cloudconformity.com/conformity-rules/CloudTrail/cloudtrail-enabled.htm)    | Ensure CloudTrail API logging is activated for all Regions |
+|  CloudTrail | [CT-001](https://www.cloudconformity.com/conformity-rules/CloudTrail/cloudtrail-global-services-enabled.html)    | Ensure CloudTrail API logging is activated for all Regions |
```

Comment:
CT-001 (CloudTrail Enabled) Rule's correct link is:
[https://www.cloudconformity.com/conformity-rules/CloudTrail/cloudtrail-enabled.html](https://www.cloudconformity.com/conformity-rules/CloudTrail/cloudtrail-enabled.html)

Reply:
hmm, thought I'd tried that one :)

Will update.